tests/yast2_gui/yast2_control_center.pm

```diff
@@ -182,7 +190,7 @@ sub start_service_manager {
     search('service');
     assert_and_click 'yast2_control-center_service-manager';
     # for a short moment the screen is not dimmed down
-    wait_still_screen 30;
+    wait_still_screen 60;
```

Comment:
That is way too long and also `wait_still_screen 60` would for sure cover the time when the minute display in the desktop clock changes so it can never pass successfully